doctr/transforms/modules.py

```diff
@@ -151,28 +154,31 @@ class InvertColorize(NestedObject):
 
     Args:
         min_val: range [min_val, 1] to colorize RGB pixels
-
+        p: probability to apply transformation
     """
-    def __init__(self, min_val: float = 0.6) -> None:
+    def __init__(self, min_val: float = 0.6, p: float = .5) -> None:
         self.min_val = min_val
-        self.togray = ToGray()
+        self.togray = ToGray(p=1.)
+        self.p = p
 
     def extra_repr(self) -> str:
         return f"min_val={self.min_val}"
 
     def __call__(self, img: tf.Tensor) -> tf.Tensor:
-        gray = self.togray(img)  # Convert to gray
-        # Random RGB shifts
-        shift_shape = [img.shape[0], 1, 1, 3] if img.ndim == 4 else [1, 1, 3]
-        rgb_shift = tf.random.uniform(shape=shift_shape, minval=self.min_val, maxval=1)
-        colorized = tf.multiply(gray, rgb_shift)
-        # Invert values
-        inverted = tf.ones_like(colorized) - colorized
-        return inverted
+        if random.random() < self.p:
+            gray = self.togray(img)  # Convert to gray
+            # Random RGB shifts
+            shift_shape = [img.shape[0], 1, 1, 3] if img.ndim == 4 else [1, 1, 3]
+            rgb_shift = tf.random.uniform(shape=shift_shape, minval=self.min_val, maxval=1)
+            colorized = tf.multiply(gray, rgb_shift)
+            # Invert values
+            inverted = tf.ones_like(colorized) - colorized
+            return inverted
+        return img
```

Comment:
Same here, the `min_val` is already a good control param, we'll add wrappers like `OneOf`